spec/lib/formatter_spec.rb

```diff
@@ -126,8 +126,8 @@ module CodeClimate::TestReporter
         end
 
         it 'should include the path prefix if set' do
-          expect(formatter.short_filename('file1')).to eq('custom/file1')
-          expect(formatter.short_filename("#{::SimpleCov.root}/file1")).to eq('custom/file1')
+          expect(formatter.send(:short_filename, 'file1')).to eq('custom/file1')
+          expect(formatter.send(:short_filename, "#{::SimpleCov.root}/file1")).to eq('custom/file1')
```

Comment:
I didn't understand before why you needed to change this spec  -- now I see it's because of the `private` keyword. 

Generally however, it's a bit of an anti-pattern to test private methods. Do you just want to leave the method public for now and leave the spec alone? It's either that, or refactor in order to test this as a public method on a different object.